test/unit_inbroker_non_parallel_SUITE.erl

```diff
@@ -338,7 +338,7 @@ log_management_during_startup1(_Config) ->
     application:unset_env(lager, extra_sinks),
     ok = try rabbit:start() of
         ok -> exit({got_success_but_expected_failure,
-                    log_rotatation_parent_dirs_test})
+                    log_rotation_parent_dirs_test})
```

Comment:
I'm vaguely concerned about these changes, since I really don't understand erlang, but afaict, they're just tokens and they aren't actually tied to anything else.